### DISCUSSION TITLE
Anyone using secondary detection methods?

### DISCUSSION THREAD

**Discussion by morrowwm** [Motion-Project/motionplus#156] [General]
I'm interested in trying one of the secondary_detection methods, but I don't see any existing discussion about them.
```
secondary_method
Values: haar, hog, dnn | Default: Not Defined
The secondary detection method.
```
I understand the instructions here
[https://motion-project.github.io/motionplus_examples.html#haar_model_training](https://motion-project.github.io/motionplus_examples.html#haar_model_training)
in general. But has anyone actually tried this and been successful?

**Comment by gabbymouth** [comment]
If you search this Discussion's forum for "Supported DNN Models" you'll see that some of us have been successful.

**Reply by morrowwm** [reply]
Thanks, @gabbymouth  
It is #82 for those looking for the same.
